experimental/PropertyDDS/packages/property-inspector-table/webpack.config.js

```diff
@@ -0,0 +1,82 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const fluidRoute = require("@fluid-tools/webpack-fluid-loader");
+const path = require("path");
+const { merge } = require("webpack-merge");
+const webpack = require("webpack");
+
+module.exports = env => {
```

Comment:
Do we need this file? Shouldn't we only need one for testing and one for the svgs?